libpod/runtime.go

```diff
@@ -739,11 +742,42 @@ func getLockManager(runtime *Runtime) (lock.Manager, error) {
 	return manager, nil
 }
 
+// probeConmon calls conmon --version and verifies it is a new enough version for
+// the runtime expectations podman currently has
+func probeConmon(conmonBinary string) error {
+	cmd := exec.Command(conmonBinary, "--version")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return err
+	}
+	r, err := regexp.Compile(`^conmon version (?P<Major>\d+).(?P<Minor>\d+).(?P<Patch>\d+)`)
+	if err != nil {
+		return errors.Wrapf(err, "conmon version changed format")
+	}
+	matches := r.FindStringSubmatch(out.String())
+	major, err := strconv.Atoi(matches[1])
```

Comment:
Should probably ensure len(matches) is what we expect so we don't segfault

Reply:
I totally goofed understanding the error handling here...

Reply:
fixed